.gitignore

```diff
@@ -112,4 +112,4 @@ trash/
 experiments/
 
 *.zip
-
+*.csv
```

Comment:
:+1:

Reply:
Hmmm, seems a bit violent to ignore all `.csv` files. We might have some in `/saved_experiments` for example

Reply:
Indeed, @aygalic could you move this in another gitignore file in the notebook experiment folder instead, please?